src/EA.Weee.DataAccess/StoredProcedure/StoredProcedures.cs

```diff
@@ -585,5 +585,20 @@ public async Task<List<AatfReuseSitesData>> GetAllAatfReuseSitesCsvData(int comp
                     complianceYearParameter, authorityParameter, panAreaParameter)
                 .ToListAsync();
         }
+
+        public async Task<List<AatfAeDetailsData>> GetAatfAeDetailsCsvData(int complianceYear, int facilityType, Guid? authority, Guid? area, Guid? panArea)
+        {
+            SqlParameter complianceYearParameter = new SqlParameter("@ComplianceYear", complianceYear);
+            SqlParameter facilityTypeParameter = new SqlParameter("@FacilityType", facilityType);
+            SqlParameter authorityParameter = new SqlParameter("@CA", (object)authority ?? DBNull.Value);
+            SqlParameter panAreaParameter = new SqlParameter("@PanArea", (object)panArea ?? DBNull.Value);
```

Comment:
Can you add some integration tests, something similar to GetAatfSubmissionsTests